HackIllinois/UI/TableView/Cells/HIBubbleCell/HIProjectCell.swift

```diff
@@ -81,13 +135,18 @@ extension HIProjectCell {
         contentStackViewHeight += titleLabel.intrinsicContentSize.height
         lhs.contentStackView.addArrangedSubview(titleLabel)
         let roomLabel = HILabel(style: .description)
-        roomLabel.text = "Table #" + String(rhs.number) //Update in UI: Phrasing not finalized
+        roomLabel.text = "Table #" + String(rhs.number)
         contentStackViewHeight += roomLabel.intrinsicContentSize.height + 3
         lhs.contentStackView.addArrangedSubview(roomLabel)
         let locationLabel = HILabel(style: .description)
-        locationLabel.text = "Meeting Room: " + rhs.room //Update in UI: Phrasing not finalized
+        locationLabel.text = "Meeting Room: " + rhs.room
```

Comment:
Does this mean it’s finalized? I noticed you deleted the comment

Reply:
Pending design's response, but we can merge it for now